### DISCUSSION TITLE
I wan't to upgrade from tailwind v2 to v3 in my existing react project but i can't . I followed all the steps from the official docs but still when i run my app no styles are applied.

### DISCUSSION THREAD

**Discussion by sachinit254** [tailwindlabs/tailwindcss#6587] [Help]
Here is my package.json file:
{
  "name": "book-app",
  "version": "0.1.0",
  "private": true,
  "proxy": "http://127.0.0.1:5000/",
  "dependencies": {
    "@emotion/react": "^11.4.1",
    "@emotion/styled": "^11.3.0",
    "@fortawesome/fontawesome-svg-core": "^1.2.36",
    "@fortawesome/free-solid-svg-icons": "^5.15.4",
    "@fortawesome/react-fontawesome": "^0.1.16",
    "@material-ui/core": "^4.12.3",
    "@material-ui/icons": "^4.11.2",
    "@material-ui/lab": "^4.0.0-alpha.60",
    "@mui/icons-material": "^5.0.1",
    "@mui/material": "^5.0.2",
    "@mui/styles": "^5.0.1",
    "@testing-library/jest-dom": "^5.14.1",
    "@testing-library/react": "^11.2.7",
    "@testing-library/user-event": "^12.8.3",
    "and": "0.0.3",
    "axios": "^0.21.4",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-redux": "^7.2.5",
    "react-router-dom": "^5.2.0",
    "react-scripts": "4.0.3",
    "redux": "^4.1.1",
    "redux-devtools-extension": "^2.13.9",
    "redux-thunk": "^2.3.0",
    "web-vitals": "^1.1.2"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "autoprefixer": "^10.4.0",
    "postcss": "^8.4.5",
    "tailwindcss": "^3.0.6"
  }
}

**Comment by lukewarlow** [comment]
Create React App 4.0 doesn't support Postcss 8 which is a requirement for Tailwind 3. If you're comfortable running pre-stable software I recommend updating to the CRA v5 alpha release which should fix your issues. Alternatively, you might have to wait till the CRA team release v5.

Alternatively, you could look into processing Tailwind using the CLI, which I believe is the approach Adam uses [here](https://github.com/adamwathan/cra-tailwind-3).

**Reply by sachinit254** [reply]
Okay thanks 😊

**Comment by saeedjabbar** [comment]
Update "react-scripts": "4.0.3" to "react-scripts": "5.0.0" and then run npm install. That should do the trick.
